src/taco-cli/cli/utils/projectHelper.ts

```
@@ -271,7 +263,7 @@ class ProjectHelper {
                 }
             });
         } catch (error) {
-            console.log(error);
```

Why not logger.logError here? do we want it to be a silent failure?